File: problem_1/problem_1.py
def multiples(n):
    sum = 0
    for i in range(n):
        if i % 3 == 0 or i % 5 == 0:
            sum += i
    return sum

print(multiples(1000))

def mult(n):
    return sum(list(filter(lambda x: x % 3 == 0 or x % 5 == 0, range(n))))

print(mult(1000))

File: problem_1/problem_1.rs
// compile with rustc problem_1.rs
// run with ./problem_1

fn main() {
    let multiples = (1..1000)
        .filter(|x| x % 3 == 0 || x % 5 == 0)
        .fold(0, |a, b| a + b);
    println!("{}", multiples)
}